schemas/public/onpremises-kfd-v1alpha2.json

```diff
@@ -1874,10 +1877,87 @@
       "then": {
         "required": [
           "type",
-          "velero"
+          "velero",
+          "etcdBackup"
         ]
       }
     },
+    "Spec.Distribution.Modules.Dr.EtcdBackup": {
+      "type": "object",
+      "description": "Configuration for the ETCD backup package.",
+      "additionalProperties": false,
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "s3",
+            "pvc",
+            "none",
+            "all"
+          ],
+          "description": "The type of the etcd backup to enable, options are:\n- `none`: no etcd backup CronJob will be installed and no etcd backup will be performed.\n- `s3`: the etcd-backup-s3 package will be enabled. It will deploy a CronJob which continuously snapshots a healthy etcd node and will save the backups in a configured S3 bucket.\n- `pvc`: the etcd-backup-pvc package will be enabled. It will deploy a CronJob which continuously snapshots a healthy etcd node and will save the backups in a configured PersistentVolumeClaim.\n- `all`: both kinds of backups will be enabled.\n\nDefault is `none`."
+        },
+        "backupPrefix": {
+          "type": "string",
+          "additionalProperties": false,
+          "description": "A prefix to be prepended to the backup filenames. If unset, the prefix defaults to the cluster's name."
+        },
+        "persistentVolumeClaim": {
```

Comment:
we are missing the required fields for persistentVolumeClaim, if a user wants to enable this, they must set the claimName, the rest of the fields can be left to their default value or specified if wanted.